src/panels/Scene/SceneTree/SceneTree.tsx

```diff
@@ -33,6 +34,11 @@ export function SceneTree() {
     sceneTreeFilterDefaults
   );
 
+  const { allowedKeys, toggleKey, selectedKeys } = useKeySettings<SceneTreeNodeData>({
+    guiPath: false,
+    label: true
```

Comment:
Reorder so the default enabled one is always first? 

Also, here I think we should make it so that the UI says "Name" instead of Label. That will be a lot less confusing to the user.

Reply:
I ordered them alphabetically which makes sense to me. I think both ways are fine in terms of ordering but we should probably stay consistent and I think alphabetically is fine since it is less complex than ordering first 1) true / false 2) alphabetically. Makes it easier to change too and I don't really see that it is an issue